### ISSUE TITLE
Installing a Package

### ISSUE THREAD

**Issue by dukesnuz** [susanBuck/dwa15-fall2017#115]
Last night I installed a package on Foobooks. I am planning on using a package for my P3, mostly for practice with using packages.  And I wanted to see how the package  worked using foobooks as practice. 

Here is what happened:

1. Installed the package on local and it worked fine.
2. Installed (git pull and ran composer) on production and received and error that curl needed to be installed. 
3. Found a similar issue on stack overflow and I installed curl
4. On production I got the Whoops page and and error in the console. Maybe error 500.
5. Put it to rest for the day.
6. This morning without performing any trouble shooting,  production seems to be working fine.

I am curious to know why production was not working and is now working.  Any thoughts are welcomed.

**Comment by gregorycoleman** [comment]
Without knowing more, it is hard to tell what actually happened.

**Comment by peggyo** [comment]
Could it have been a cache problem yesterday? Or a server restart that was missed yesterday but done today?

**Comment by dukesnuz** [comment]
@peggyo Yes I was thinking the same thing - a server restart. I did restart the server when I was trouble shooting.  I am not sure if Digital Ocean performs some sort of maintenance at night.  Then again it could have been a cache issue -  good thought.  I did learn that if I have this issue again, I will do minor trouble shooting and if possible wait until the next day so as not to  waste time trouble shooting.

I am going to install the same package in my P3 and if I have the same issue, I will take a screen shot and post it.
